packages/react/src/components/Feedback/PackageRuleContribution.tsx

```diff
@@ -48,11 +51,16 @@ const PackageRuleContributionInternal: React.FC<Props> = ({ source }) => {
   const nextPagePath = path.dirname(location.pathname);
 
   const [email, setEmail] = useState(window.electron.getState("email"));
+  const [showEmailModal, setShowEmailModal] = useState(!isEmailSet());
   const [packageName, setPackageName] = useState("");
   const [packageVersion, setPackageVersion] = useState("");
+  const [packageError, setPackageError] = useState("");
```

Comment:
Hi instead of using this versionError and packageError store everything in local, just return them from your check function.{true, ""}, {false, "error message"}

Reply:
I use these state variables to highlight specific form fields that the customer needs to correct. If I do not use state variables, I am unsure how to mark the error in the AWS UI elements. Should I just return and display a generic error message?